charts/bindplane/templates/bindplane.yaml

```diff
@@ -368,7 +368,7 @@ spec:
         {{- if eq (include "bindplane.deployment_type" .) "StatefulSet" }}
         {{- if and (.Values.prometheus.enable) (.Values.prometheus.enableSideCar) }}
         - name: prometheus
-          image: prom/prometheus:v2.47.2
+          image: ghcr.io/observiq/bindplane-prometheus:{{ include "bindplane.tag" . }}
```

Comment:
We use `{{ include "bindplane.tag" . }}` to determine the bindplane image tag as well. This will allow us to always deploy the correct Prometheus image for a given release.